go/vt/mysqlctl/backup.go

```diff
@@ -116,6 +113,10 @@ func registerBackupFlags(fs *pflag.FlagSet) {
 // - shuts down Mysqld during the backup
 // - remember if we were replicating, restore the exact same state
 func Backup(ctx context.Context, params BackupParams) error {
+	if params.Stats == nil {
+		params.Stats = backupstats.DefaultBackupStats()
```

Comment:
The whole approach of this PR is to bundle up stats in an interface that can be passed downwards into deeper parts of the backup system, rather than having different engines and plugins maintain their own stats or import a set of global stats.

Having `Stats` be part of `BackupParams` means that vtbackup could pass in different metrics than other packages that use `mysqlctl`. Also will be nice because we can pass in mock stats to make testing easy.